docs/library/machine.I2C.rst

```diff
@@ -79,18 +79,17 @@ The following methods implement the primitive I2C master bus operations and can
 be combined to make any I2C transaction.  They are provided if you need more
 control over the bus, otherwise the standard methods (see below) can be used.
 
+These methods are available on software I2C. On ESP8266 they are also available
+on hardware I2C.
```

Comment:
The esp8266 doesn't have hardware I2C, it just has `machine.I2C` which is always a software implementation.